httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/HttpRFC7578Multipart.java

```diff
@@ -129,7 +128,7 @@ public byte[] encode(final byte[] bytes) {
             return buffer.toByteArray();
         }
 
-        public byte[] decode(final byte[] bytes) throws DecoderException {
+        public byte[] decode(final byte[] bytes) throws IllegalArgumentException {
```

Comment:
Since this class is package scoped, there won't be any external code expecting this exception

Reply:
Signature

Reply:
No runtime exceptions in the signature